### ISSUE TITLE
NotFoundException in Group Folder

### ISSUE THREAD

**Issue by metheos** [nextcloud/recognize#706]
### Which version of recognize are you using?

3.5.0

### Enabled Modes

Object recognition, Face recognition

### TensorFlow mode

Normal mode

### Which Nextcloud version do you have installed?

25.0.4

### Which Operating system do you have installed?

Ubuntu 22.04.2

### Which Docker container are you using to run Nextcloud? (if applicable)

N/A

### How much RAM does your server have?

12GiB

### What processor Architecture does your CPU have?

x86_64

### Describe the Bug

Error when classifying a file in a group folder.

### Expected Behavior

Processing of classification should finish without error.

### To Reproduce

Create a group folder and share it with at least one user. 
Put photo into group folder. 
run occ recognize:classify

### Debug log

Classifying array (
  0 => '/tmp/oc_tmp_fHzB3h-.jpg',
  1 => '/tmp/oc_tmp_agvAtr-.jpg',
  2 => '/tmp/oc_tmp_aqV6dK-.jpg',
  3 => '/tmp/oc_tmp_VLmyLN-.jpg',
  4 => '/tmp/oc_tmp_Wt93UX-.jpg',
  5 => '/tmp/oc_tmp_ohn1Rt-.jpg',
  6 => '/tmp/oc_tmp_s4hwtA-.jpg',
)
Running array (
  0 => '/var/www/nextcloud/apps/recognize/bin/node',
  1 => '/var/www/nextcloud/apps/recognize/src/classifier_imagenet.js',
  2 => '-',
)
Classifier process output: 2023-03-01 15:13:07.615478: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

Classifier process output: {
  className: 'photocopier',
  probability: 0.7441722750663757,
  rule: { label: 'office', threshold: 0.1 }
}

Classifier process output: {
  className: 'printer',
  probability: 0.1145385354757309,
  rule: { label: 'office', threshold: 0.1 }
}

Classifier process output: {
  className: 'polaroid camera',
  probability: 0.0007486474933102727,
  rule: { label: 'camera', threshold: 0.1, categories: [ 'photography' ] }
}

Classifier process output: {
  className: 'overskirt',
  probability: 0.0007342754979617894,
  rule: { label: 'portrait', threshold: 0.08, categories: [ 'people' ] }
}

Classifier process output: {
  className: 'cash machine',
  probability: 0.0007029475527815521,
  rule: { threshold: 0.1, priority: -2 }
}

Classifier process output: {
  className: 'space bar',
  probability: 0.0006621112115681171,
  rule: {
    label: 'keyboard',
    threshold: 0.5,
    context: 'screen',
    categories: [ 'computer' ]
  }
}

Classifier process output: {
  className: 'black-footed ferret',
  probability: 0.0006103127961978316,
  rule: { label: 'animal', context: 'nature', threshold: 0.1 }
}

Result for /EC2DA82F-F3FC-4492-8864-F2D6EF517D60/files/GroupFolderTest/product images/Lexmark-xc4140-xc4150-image.jpg(oc_tmp_fHzB3h-.jpg) = ["Office"]
Classifier process output: {
  className: 'photocopier',
  probability: 0.8550654053688049,
  rule: { label: 'office', threshold: 0.1 }
}

Classifier process output: {
  className: 'printer',
  probability: 0.035257626324892044,
  rule: { label: 'office', threshold: 0.1 }
}
{
  className: 'black-footed ferret',
  probability: 0.000581808271817863,
  rule: { label: 'animal', context: 'nature', threshold: 0.1 }
}
{
  className: 'polaroid camera',
  probability: 0.0005186334601603448,
  rule: { label: 'camera', threshold: 0.1, categories: [ 'photography' ] }
}
{
  className: 'typewriter keyboard',
  probability: 0.0004904744564555585,
  rule: { label: 'keyboard', threshold: 0.1 }
}
{
  className: 'cash machine',
  probability: 0.0004573196347337216,
  rule: { threshold: 0.1, priority: -2 }
}
{
  className: 'orange',
  probability: 0.0004214751534163952,
  rule: { label: 'fruit', threshold: 0.1, categories: [ 'food' ] }
}

Result for /EC2DA82F-F3FC-4492-8864-F2D6EF517D60/files/GroupFolderTest/product images/XM5365-XM5370-Right.jpg(oc_tmp_agvAtr-.jpg) = ["Office"]
Classifier process output: {
  className: 'photocopier',
  probability: 0.8759931921958923,
  rule: { label: 'office', threshold: 0.1 }
}

Result for /EC2DA82F-F3FC-4492-8864-F2D6EF517D60/files/GroupFolderTest/product images/e4515ac.png(oc_tmp_aqV6dK-.jpg) = ["Office"]

In Root.php line 209:

  [OCP\Files\NotFoundException]
  /EC2DA82F-F3FC-4492-8864-F2D6EF517D60/files/GroupFolderTest

Exception trace:
  at /var/www/nextcloud/lib/private/Files/Node/Root.php:209
 OC\Files\Node\Root->get() at /var/www/nextcloud/lib/private/Files/Node/Node.php:293
 OC\Files\Node\Node->getParent() at /var/www/nextcloud/lib/private/Share20/Manager.php:1747
 OC\Share20\Manager->getAccessList() at /var/www/nextcloud/lib/private/Share20/ShareHelper.php:52
 OC\Share20\ShareHelper->getPathsForAccessList() at /var/www/nextcloud/apps/systemtags/lib/Activity/Listener.php:187
 OCA\SystemTags\Activity\Listener->mapperEvent() at /var/www/nextcloud/apps/systemtags/lib/AppInfo/Application.php:74
 OCA\SystemTags\AppInfo\Application->OCA\SystemTags\AppInfo\{closure}() at /var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php:251
 Symfony\Component\EventDispatcher\EventDispatcher->callListeners() at /var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php:73
 Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at /var/www/nextcloud/lib/private/EventDispatcher/EventDispatcher.php:88
 OC\EventDispatcher\EventDispatcher->dispatch() at /var/www/nextcloud/lib/private/EventDispatcher/SymfonyAdapter.php:99
 OC\EventDispatcher\SymfonyAdapter->dispatch() at /var/www/nextcloud/lib/private/SystemTag/SystemTagObjectMapper.php:174
 OC\SystemTag\SystemTagObjectMapper->assignTags() at /var/www/nextcloud/apps/recognize/lib/Service/TagManager.php:77
 OCA\Recognize\Service\TagManager->assignTags() at /var/www/nextcloud/apps/recognize/lib/Classifiers/Images/ImagenetClassifier.php:48
 OCA\Recognize\Classifiers\Images\ImagenetClassifier->classify() at /var/www/nextcloud/apps/recognize/lib/Command/Classify.php:127
 OCA\Recognize\Command\Classify->execute() at /var/www/nextcloud/apps/libresign/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php:1040
 Symfony\Component\Console\Application->doRunCommand() at /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at /var/www/nextcloud/lib/private/Console/Application.php:213
 OC\Console\Application->run() at /var/www/nextcloud/console.php:100
 require_once() at /var/www/nextcloud/occ:11

**Comment by marcelklehr** [comment]
This doesn't seem to be a problem with recognize. Could you please post over here https://github.com/nextcloud/groupfolders/issues/2301 the details of your setup that I wasn't able to provide?

**Comment by metheos** [comment]
> This doesn't seem to be a problem with recognize. Could you please post over here [nextcloud/groupfolders#2301](https://github.com/nextcloud/groupfolders/issues/2301) the details of your setup that I wasn't able to provide?

Done!

**Comment by marcelklehr** [comment]
I cannot reproduce this anymore. I'm closing this for now. If you still have this issue, you can comment below and I will reopen.
